setup.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash		
+ 		
+ # Shell script to start the database and app services before running the tests.		
+ 		
+ ## color codes		
+ RED='\033[1;31m'		
+ GREEN='\033[1;32m'		
+ YELLOW='\033[1;33m'		
+ CYAN='\033[1;36m'		
+ PLAIN='\033[0m'		
+ 		
+ ## variables		
+ HOST=localhost		
+ USER='admin'		
+ PASSWORD='pass'		
+ PORT=8080		
+ DATABASE='testdb'		
+ if [ "$1" ]; then		
+     HOST=$1		
+ fi		
+ if [ "$2" ]; then		
+     PORT=$2		
+ fi		
+ if [ "$2" ]; then		
+     USER=$3		
+ fi		
+ if [ "$4" ]; then		
+     PASSWORD=$4		
+ fi		
+ if [ "$5" ]; then		
+     DATABASE=$5		
+ fi		
+ 		
+ ## check if docker exists		
+ printf "\n${RED}>> Checking for docker${PLAIN} ${GREEN}...${PLAIN}"		
+ docker -v > /dev/null 2>&1		
+ DOCKER_EXISTS=$?		
```

Comment:
Check for `docker-compose` as well since we use it in this script or remove the dependency on it and this can be the same.